config/checkstyle/suppressions.xml

```diff
@@ -6,4 +6,6 @@
 
 <suppressions>
   <suppress checks="JavadocType" files=".*Test\.java"/>
+  <suppress checks="JavadocMethod" files=".*[\\/]src[\\/]test[\\/]"/>
+  <suppress checks="JavadocMethod" files=".*[\\/]src[\\/]systemtest[\\/]"/>
```

Comment:
Is it possible to suppress the JavadocMethod check for functions with the `@Test` annotation present? The `@Before`, `@After` and other test-related annotations can also be suppressed.

Reply:
@yhtMinceraft1010X

Reply:
The check is already suppressed for the test-related annotations under `MissingJavadocMethod` in the `checkstyle.xml` file. The reason why I put these suppressions is to allow for missing `@throws` tags in the non-annotated functions for test code since my proposed style guide only calls for `@throws` tags in main code.

Reply:
I see, okay sure. I think it should be sufficient for now, although I think in the future we can try to generalize the style guide to cover all aspects of the code base and to maintain consistency.